packages/babel6/__tests__/babel6.test.js

```diff
@@ -1,15 +1,20 @@
 import test from 'ava'
+import sinon from 'sinon'
 import babel6 from '../index'
 
 test('Babel default options work', (t) => {
   const block = babel6()
 
+  const addLoader = sinon.spy((loaderDef) => config => config)
   const context = {
     fileType: () => '*.js'
   }
 
-  t.deepEqual(block(context), {})
-  t.deepEqual(block.post(context).module.loaders, [
+  t.deepEqual(block(context, { addLoader })({}), {})
+  t.deepEqual(block.post(context, { addLoader })({}), {})
```

Comment:
Btw, this is a good video about spies vs mocks vs not using them https://www.youtube.com/watch?v=URSWYvyc42M